modules/combat.py

```diff
@@ -385,12 +390,27 @@ def clear_map(self):
             Logger.log_warning("Fleet lock is not supported, disabling it.")
             Utils.wait_update_screen()
 
-        #swipe map to fit everything on screen
+        # swipe map to fit everything on screen
         def swipe(cmap):
             if cmap == 'E-C3' or cmap == 'E-A3':
                 Utils.swipe(960, 800, 960, 400, 100)
+            elif cmap == '11-3':
+                Utils.swipe(1020, 570, 1300, 540, 100)
+            elif cmap == '12-2':
+                Utils.swipe(1000, 570, 1300, 540, 100)
+            elif cmap == '12-3':
+                Utils.swipe(1250, 530, 1300, 540, 100)
+            elif cmap == '13-1':
+                Utils.swipe(1020, 500, 1300, 540, 100)
+            elif cmap == '13-2':
+                Utils.swipe(1125, 550, 1300, 540, 100)
+            elif cmap == '13-3':
+                Utils.swipe(1150, 510, 1300, 540, 100)
+            elif cmap == '13-4':
+                Utils.swipe(1200, 460, 1300, 540, 100)
```

Comment:
As @Al12rs said, this could be better implemented by using a dictionary, I originally made it an `if` because it was only for a single map.